client/src/routes/app/[country]/[map]/[year]/initialize/LoadRegions.ts

```diff
@@ -4,8 +4,9 @@ import { RegionsStore } from '$lib/stores/Regions';
 import { TossupCandidateStore, SelectedCandidateStore } from '$lib/stores/Candidates';
 import { EditRegionModalStore } from '$lib/stores/Modals';
 import type Region from '$lib/types/Region';
+import { InteractionStore } from '$lib/stores/Interaction';
 
-function fillRegion(regionID: string) {
+function fillRegion(regionID: string, increment?: boolean) {
```

Comment:
I think in general, optional parameters should be avoided. It's fine for now, it just becomes a hassle to do more checks for possible undefined values as a project grow.

Reply:
I am in agreement, did not see another way to do it here. Breaking it out into a function is less desirable, and making it a required parameter requires changes to all the other calls, which we could do.